src/conf.py

```diff
@@ -16,6 +16,16 @@
 import sys
 import shutil
 
+# Add python modules to be documented
+python_root = './coreComponents/python/modules'
+python_modules = ['geosx_mesh_tools_package',
```

Comment:
BTW, you can use `tuples` instead of list if you do not want the list to be extended.